setup.cfg

```diff
@@ -64,6 +67,8 @@ testing =
     pytest-cov
 
 [options.entry_points]
+console_scripts =
```

Comment:
extra: adding a console script for calling the channels-last transformation would also be useful

Reply:
Yeah, that is nice to have. I added support for it in the most recent push.